Omise.Tests/Resources/ChargeResourceTest.cs

```diff
@@ -2,66 +2,65 @@
 using Omise.Resources;
 using Omise.Models;
 using System.Collections.Generic;
-using System.Threading.Tasks;
-
-namespace Omise.Tests.Resources
+using System.Threading.Tasks;
+namespace Omise.Tests.Resources
 {
     [TestFixture]
-    public class ChargeResourceTest : ResourceTest<ChargeResource>
+    public class ChargeResourceTest : ResourceTest<ChargeResource>
     {
         const string ChargeId = "chrg_test_4yq7duw15p9hdrjp8oq";
 
         [Test]
-        public async Task TestGetList()
+        public async Task TestGetList()

```

Comment:
It seems like there's no actual change here so can we exclude these false changes?

Reply:
@AnasNaouchi why do these lines show a change when there is no actual change?

Reply:
There is no change but github keeps showing a change

Reply:
I think it is end of line sequence (LF vs CRLF) issue. You can change it in vscode footer. May be you switch to different one by mistake.

Reply:
```
git config diff.wsErrorHighlight all
git diff
```